src/index.html

```diff
@@ -211,6 +216,10 @@
       document.querySelector('progress-bar').removeAttribute('hidden');
       window.location.href += `screenshot/${url.value.replace('?', '%3F')}?width=${window.innerWidth}&height=${window.innerHeight}`;
     }
+
+    function onEmptyUrl() {
+      document.getElementById('url').style.borderColor = '#FF9999';
```

Comment:
Let's change this to use a class instead so the style can be kept in CSS.

Reply:
Done